<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowTest</class>
 <widget class="QMainWindow" name="MainWindowTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlightPath</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="leftPanel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="algoBox" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Algorithme :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="algoCombobox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>TSP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Breitling</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="heures" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="Heures">
          <item row="2" column="0" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Coucher du soleil :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Heure de d√©part :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_8">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Temps de ravitaillement :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Lever du soleil :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignLeft">
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSpinBox" name="couH">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>h</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="couM">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignLeft">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSpinBox" name="depH">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>h</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="depM">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignLeft">
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSpinBox" name="levH">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>h</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="levM">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="1" alignment="Qt::AlignLeft">
           <widget class="QWidget" name="horizontalWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignLeft">
              <widget class="QSpinBox" name="tmpRav">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>100</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="label_35">
               <property name="text">
                <string>min</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="choixDepartArrivee" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="departarrivee">
          <item>
           <layout class="QVBoxLayout" name="departBox">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>D√©part :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="depComboBox"/>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="1">
               <widget class="QLabel" name="depOACILabel">
                <property name="text">
                 <string>----</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>VFR nuit</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Statut</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="depStatutLabel">
                <property name="text">
                 <string>----</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Code OACI</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="depVFRLabel">
                <property name="text">
                 <string>----</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Essence</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="depEssenceLabel">
                <property name="text">
                 <string>----</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="arriveeBox">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Arriv√©e :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="arrComboBox"/>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="2" column="1">
               <widget class="QLabel" name="arrVFRLabel">
                <property name="text">
                 <string>----</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="arrStatutLabel">
                <property name="text">
                 <string>----</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Statut</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>VFR nuit</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Code OACI</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="arrOACILabel">
                <property name="text">
                 <string>----</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>Essence</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="arrEssenceLabel">
                <property name="text">
                 <string>----</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="boucleBox">
            <item alignment="Qt::AlignTop">
             <widget class="QCheckBox" name="boucle">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Boucle</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="vitesse_capacite_conso" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="consommation"/>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="vitesse"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Vitesse :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="capacite"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Capacit√© :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Consommation :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>kt</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>L</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>L/h</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="essenceView">
         <item>
          <widget class="QLabel" name="essenceViewLabel">
           <property name="text">
            <string>Essence :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="essenceViewTable"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="VFRView">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>VFR nuit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_3"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="computeButton">
         <property name="text">
          <string>Calculer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="excelTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir...</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
